workflows/sarek.nf

```diff
@@ -1308,6 +1308,13 @@ def flowcellLaneFromFastq(path) {
     return fcid
 }
 
+// Check the parameters tools or skip_tools, then compare it against the provided tool
+// Returns true/false based on whether 'tool' is found in 'parameter'
+def checkTools(parameter, tool) {
+    parameter.split(',').any{ it.toLowerCase().contains(tool) }
```

Comment:
How about using `equals` instead of `contains`?  In this example, `X` is `true` and wrongly indicate that `manta` is among the requested tools. 

```
tools = 'strelka,manta2'
X = tools.split(',').any{ it.toLowerCase().contains('manta') }
Y = tools.split(',').any{ it.toLowerCase().equals('manta') }
```
The variable `Y`, on the other hand, is `false` and correctly indicate that `manta` is not among the requested tools. (I know we don't have a "manta2", but you get the point.)

Also, if you go with `equals()` then I also suggest that something the like Python-function `.strip()` is called before the call to `equals()`. The reason being that the user might add a space in the list of tools - like this:
```
tools = 'strelka, manta'
```